packages/admin/src/pages/super-admin-dashboard/user/[id]/index.page.tsx

```diff
@@ -41,7 +43,11 @@ const UserPage = (user: User) => {
                 <h2 className="govuk-heading-m">User Information</h2>
                 <SummaryList
                   rows={[
-                    { key: 'Account created', value: '', action: <></> },
+                    {
+                      key: 'Account created',
+                      value: creationDate,
+                      action: <></>,
+                    },
```

Comment:
I think we only want to render this row when we have a `creationDate`

Reply:
Spoke to Matt about this one, we're going with a dash when there's no data to display (adding that in now)